install/MacOSXcomm.html

```diff
@@ -49,7 +49,7 @@
 <!-- /Header -->
 
   <div class="nomenu" id="mBody">
-    <div id="mainContent">
```

Comment:
I think the “div mainContent” here is part of the CSS formatting of the page. (I’ve seen lots of pages missing the corresponding close div)

Reply:
Done